### ISSUE TITLE
[Bug] consider increasing contrast of "search language" placeholder text

### ISSUE THREAD

**Issue by asadotzler** [mozilla-mobile/fenix#9728]
## Steps to reproduce
Navigate to Settings -> Language
Inspect search field for "search language" placeholder text and notice low contrast gray text on gray background

### Expected behavior
higher contrast text over background

### Actual behavior
lower contrast text over background

### Device information

* Android device: Galaxy S10e
* Fenix version: 4/3 Nightly

**Comment by kbrosnan** [comment]
@brampitoyo does this text conform to AA or AAA for text contrast?

**Comment by brampitoyo** [comment]
@asadotzler @kbrosnan Unfortunately, our placeholder texts don’t pass WCAG 2.0 AA or AAA.

Changing the colour of this placeholder text will impact how the URL is displayed inside the address bar and search screen, so that’s a bigger conversation that we should have with @AmyYLee and @shorlander.

In the meantime, we have two options to fix the search box inside Language settings:

1. Keep the box the same colour (fill Grey Light 30 `#e0e0e6`), but darken the placeholder text so it’s the same colour as the rest of the text (On-Light Ink 80 `#202340`)

<img width="360" src="https://user-images.githubusercontent.com/51007/78733188-d85a2e00-7998-11ea-9e09-7ed6b7a716e8.png">

---

2. Lighten the box (fill White `#ffffff`, border Grey Light 30 `#e0e0e6`), but keep the placeholder text the same colour (Grey Light 90 `#80808e`)

<img width="360" src="https://user-images.githubusercontent.com/51007/78733297-25d69b00-7999-11ea-9ec4-d61b8a927adf.png">

---

Either way, let’s make sure that the placeholder text size is the same as the text size we use for labels (16dp).

Any thoughts?

**Comment by AmyYLee** [comment]
"Keep the box the same colour (fill Grey Light 30 #e0e0e6), but darken the placeholder text so it’s the same colour as the rest of the text (On-Light Ink 80 #20123a)"

@brampitoyo My recommendation is for Option 1 so we keep the search bar colour consistent with the browser search bar. As for the placeholder text. Can we have this updated across the app to be Dark Grey 5 (#5B5B66). It's still a bit lighter than regular text but passes contrast test. I've updated this in the UI library.

**Comment by mcarare** [comment]
@AmyYLee For consistency, shouldn't we use the same search bar in all app searches? For example, in the saved logins there is another search field, in the "title bar" that expands.

![prtl](https://user-images.githubusercontent.com/48995920/79334854-b7aa4f00-7f29-11ea-9b28-c3257a9bd0db.png)
![prtll](https://user-images.githubusercontent.com/48995920/79334800-a2cdbb80-7f29-11ea-9546-740713ecac10.png)

As for the hint (placeholder) color Dark Grey 5 (#5B5B66) should be for all themes?

Also cc @vesta0: are we considering adding a search to history and bookmarks?

**Comment by vesta0** [comment]
@mcarare that's a good idea to consider for future, although history and bookmark items already get pulled when user starts typing in the search/url bar.

**Comment by brampitoyo** [comment]
@mcarare @AmyYLee Yes, I agree. For consistency, I think that we should use the same search bar as the one found on “Saved logins”.

The way to access this search UI is also the same: through a search magnifying glass icon on the top-right corner of the app bar.

<img width="240" src="https://user-images.githubusercontent.com/51007/79714102-e6f2fe80-8323-11ea-98e6-cd48cee25974.png"> <img width="240" src="https://user-images.githubusercontent.com/51007/79714104-e9555880-8323-11ea-9888-cff3be5ed858.png">

**Comment by sv-sdeiac** [comment]
Hi, I can confirm that the new Search field is displayed on the Language selection option.

Build:
- Firefox Preview Nightly 3/30 #21210606.

Devices:
- Google Pixel 3 (Android 10);
- Samsung Galaxy Note 8 (Android 9);
- LG G7 fit (Android 8.1).
